src/main/java/qna/domain/User.java

```diff
@@ -4,16 +4,33 @@
 
 import java.util.Objects;
 
-public class User {
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+
+@Entity
+@Table(uniqueConstraints = {
+    @UniqueConstraint(name = "UK_a3imlf41l37utmxiquukk8ajc", columnNames = {"userId"})
+})
+public class User extends BaseEntity{
     public static final GuestUser GUEST_USER = new GuestUser();
-
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+    @Column(length = 20)
     private String userId;
```

Comment:
위에 설정을 하셨지만 아래처럼 `unique` 설정을  할 수도 있겠네요 :)

```suggestion
    @Column(length = 20, nullable = false, unique = true)
    private String userId;
```

Reply:
피드백 감사합니다! 코드 수정하겠습니다 👍